Doc/library/asyncio-queue.rst

```diff
@@ -1,41 +1,41 @@
 .. currentmodule:: asyncio
 
+
+======
 Queues
```

Comment:
Perhaps rename to Queues in asyncio or Asyncio Queues

Reply:
I'd keep it as is so that it's just "Queues" in the TOC.

Reply:
:+1: